client/src/pages/HomePage.vue

```diff
@@ -17,7 +17,9 @@
             class="d-flex flex-column justify-start align-start pl-10"
           >
             <LAButton text="Get Started" width="350px" />
-            <LAButton text="I Already Have an Account" width="350px" />
+            <router-link style="text-decoration: none;" to="/login">
```

Comment:
We can also use class="text-decoration-none" from vuetify.

Reply:
Fixed